src/coffee/bootstrap.offcanvas.coffee

```diff
@@ -353,6 +353,8 @@
             oc = new Offcanvas $(this)
 
         $(window).on 'resize', ->
+          if 'text' != $(document.activeElement).attr 'type'
+            return
```

Comment:
hm.. the next two lines feature two-space indentation, at least in my IDE (VS Code):
```
$('.navbar-offcanvas.in').each ->
  $(@).height('').removeClass 'in'
```